<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgLog</class>
 <widget class="QDialog" name="DlgLog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>对话框试程序</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>716</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>当前LOG目录:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtLogPath"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>下载速度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtDownSpeed">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="inputMask">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>KB/s</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>500</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabDownLog">
       <attribute name="title">
        <string>下载Log</string>
       </attribute>
       <widget class="QGroupBox" name="groupSvrList">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>100</y>
          <width>501</width>
          <height>51</height>
         </rect>
        </property>
        <property name="title">
         <string>服务器</string>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>501</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </widget>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>50</y>
          <width>328</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="btnAnalystLog">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>分析</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDumpCmd">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>生成命令行</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnImportLog">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>导入</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMergeLog">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>合并</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>245</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnAddTask">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>加任务</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClearTask">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>清任务</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDownLogByFileName">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>开始</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>751</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>文件名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtDownByFileName"/>
         </item>
         <item>
          <widget class="QCheckBox" name="chkCompressOnFinish">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>下载完压缩</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkFuzzy">
           <property name="text">
            <string>模糊匹配</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkGmlog">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>gmlog</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>160</y>
          <width>211</width>
          <height>311</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="chkAllGroup">
           <property name="text">
            <string>AllGroup</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeView" name="tvSvr"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabCheckLog">
       <attribute name="title">
        <string>检查Log</string>
       </attribute>
       <widget class="QTreeView" name="treeViewWarnLog">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>160</y>
          <width>581</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="edtMsg">
      <property name="minimumSize">
       <size>
        <width>759</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>1000</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>edtLogPath</sender>
   <signal>editingFinished()</signal>
   <receiver>DlgLog</receiver>
   <slot>OnLogPathEditFinish()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtDownSpeed</sender>
   <signal>editingFinished()</signal>
   <receiver>DlgLog</receiver>
   <slot>OnDownSpeedEditFinish()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAnalystLog</sender>
   <signal>clicked()</signal>
   <receiver>DlgLog</receiver>
   <slot>AnalystLog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnImportLog</sender>
   <signal>clicked()</signal>
   <receiver>DlgLog</receiver>
   <slot>ImportLog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMergeLog</sender>
   <signal>clicked()</signal>
   <receiver>DlgLog</receiver>
   <slot>MergeLog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDumpCmd</sender>
   <signal>clicked()</signal>
   <receiver>DlgLog</receiver>
   <slot>OnBtnDumpCmd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeViewWarnLog</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>DlgLog</receiver>
   <slot>OnDBClickedWarnLogList(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tvSvr</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>DlgLog</receiver>
   <slot>OnSvrTreeClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddTask</sender>
   <signal>clicked()</signal>
   <receiver>DlgLog</receiver>
   <slot>OnAddTask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClearTask</sender>
   <signal>clicked()</signal>
   <receiver>DlgLog</receiver>
   <slot>OnClearTask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDownLogByFileName</sender>
   <signal>clicked()</signal>
   <receiver>DlgLog</receiver>
   <slot>DownLogByFileName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>DownTodayLog()</slot>
  <slot>AnalystLog()</slot>
  <slot>OnDBClickedWarnLogList(QModelIndex)</slot>
  <slot>DownLogByDay()</slot>
  <slot>MergeLog()</slot>
  <slot>OnLogPathEditFinish()</slot>
  <slot>ImportLog()</slot>
  <slot>DownLogByFileName()</slot>
  <slot>OnDownSpeedEditFinish()</slot>
  <slot>OnSvrGropuChanged(QString)</slot>
  <slot>OnClearTask()</slot>
  <slot>OnAddTask()</slot>
  <slot>OnBtnDumpCmd()</slot>
  <slot>OnSvrTreeClicked()</slot>
  <slot>OnBtnAllGroup()</slot>
 </slots>
</ui>
